lagom-cloud-object-storage-example/account-impl/src/test/java/com/lightbend/lagom/account/impl/AccountEntityTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.account.impl;
+
+import akka.Done;
+import akka.actor.ActorSystem;
+import akka.testkit.JavaTestKit;
+import com.lightbend.lagom.javadsl.persistence.PersistentEntity;
+import com.lightbend.lagom.javadsl.testkit.PersistentEntityTestDriver;
+import com.lightbend.lagom.javadsl.testkit.PersistentEntityTestDriver.Outcome;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AccountEntityTest {
+
+  static ActorSystem system;
+
+  @BeforeClass
+  public static void setup() {
+    system = ActorSystem.create("AccountEntityTest");
+  }
+
+  @AfterClass
+  public static void teardown() {
+    JavaTestKit.shutdownActorSystem(system);
+    system = null;
+  }
+
+  @Test
+  public void testHelloWorld() {
```

Comment:
Not much. Only the reject withdraw makes sense to split. 

You can't test `withdraw` without first doing `deposit`, so the `withdraw` test would simply repeat the operations of the `deposit` test. 

It tests the lifecycle of an account from 0 to some balance, to less balance. There is not much more functionally than that. We may have add one unnecessary deposit. 

A good improved would be calling it `testAccount` instead of `testHelloWorld` :-)